src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCell.DataGridViewCellAccessibleObject.cs

```diff
@@ -116,6 +117,19 @@ public override string DefaultAction
 
         public override AccessibleRole Role => AccessibleRole.Cell;
 
+        public int RowColumnStartIndex
+        {
+            get
+            {
+                if (LocalAppContextSwitches.DataGridViewStartRowColumnIndex)
+                {
+                    return 1;
+                }
+
+                return 0;
+            }
+        }
```

Comment:
this could be simplified 
```suggestion
        public int RowColumnStartIndex => LocalAppContextSwitches.DataGridViewStartRowColumnIndex ? 1 : 0;
        
```